integration-test/src/tc0/tc.test.ts

```diff
@@ -0,0 +1,14 @@
+import { readFileSync } from 'fs';
+import { join } from 'path';
+import DxfParser from 'dxf-json'
+
+describe('integration tc0', () => {
+  test('tc0', () => {
+    const content = readFileSync(join(__dirname, './tc.dxf'), 'utf-8');
+    const parser = new DxfParser()
+    const dxf = parser.parseSync(content)
+    const answer = JSON.parse(readFileSync(join(__dirname, './tc.json'), 'utf-8'))
```

Comment:
Our integration test is done by following procedures

1. Load `tc.dxf`
2. Parse it
3. Load `tc.json` and compare